src/backend/modules/user/resolvers.js

```diff
@@ -26,13 +28,18 @@ const resolvers = {
       return User.findOne(id);
     },
     login: async (parent, { email, password }) => {
-      let u = await User.findOne({ email: email });
+      const u = await User.findOne({ email: email });
       if (u == null) {
         throw new UserInputError("Username or Password is incorrect");
       } else {
-        let valid = await u.comparePassword(password);
+        const valid = await u.comparePassword(password);
         if (valid) {
-          return u;
+          const payload = {
+            user: {
+              id: u.id
+            }
+          };
+          return jwt.sign(payload, config.jwtSecret, { expiresIn: "1d" });

```

Comment:
Is it customary for this to expire in just 1 day? Not sure what the norms are but this feels short :open_math: